### ISSUE TITLE
no_decay = ['bias', 'LayerNorm.bias', 'LayerNorm.weight']

### ISSUE THREAD

**Issue by RayXu14** [huggingface/pytorch-pretrained-BERT#492]
what does this means? Whay these three kind no decay?

**Comment by thomwolf** [comment]
Yes. We are reproducing the behavior of the original optimizer, see [here](https://github.com/google-research/bert/blob/master/optimization.py#L65).

**Comment by RayXu14** [comment]
thanks~

**Comment by lxgend** [comment]
but why?

**Comment by fightnyy** [comment]
I have the same question, but did this prove to be better? Or is it just to speed up calculations?
